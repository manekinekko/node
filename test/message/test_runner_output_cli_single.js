// Flags: --no-warnings --test test/fixtures/test-runner/index.test.js
// FlagsOnly
'use strict';
require('../common');
