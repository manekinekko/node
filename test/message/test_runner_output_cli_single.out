TAP version 13
# Subtest: *
    # Subtest: this should pass
    ok 1 - this should pass
      ---
      duration_ms: *
      ...
    1..1
ok 1 - *
  ---
  duration_ms: *
  ...
1..1
# tests 1
# pass 1
# fail 0
# cancelled 0
# skipped 0
# todo 0
# duration_ms *
